api/chat.go

```diff
@@ -590,6 +605,11 @@ type wsIdReq struct {
 	Id uint `json:"id"`
 }
 
+type wsReactToReq struct {
+	wsIdReq
+	Emoji string `json:"emoji"`
```

Comment:
Just to be consistent with the other names
```suggestion
	Reaction string `json:"emoji"`
```